src/configs/OnboardingList.config.js

```diff
@@ -79,6 +84,23 @@ export function OnboardingListDefinition(props) {
       orders: WooCommerceSdk.orders.get
     },
     cards: [
+      {
+        name:"Explore the features included with your solution",
+        id: "mypluginsandtools",
+        text: __("Explore the features included with your solution", "wp-module-ecommerce"),
+        state: {
+          isAvailable: (queries) => queries?.settings?.hasBrandSignedUp,
+          isCompleted: (queries) => !(queries?.settings?.hasBrandSignedUp),
+          url: () =>
+            `${window.location.href.split('#')[0]}#/my_plugins_and_tools`,
+          target: () => "_blank",
```

Comment:
Hi @aratidgr8 / @searunt , can you please confirm , in the next steps when user clicks on `Explore the features included with your solution` are we expecting it to open it in new tab ??

Reply:
It will open in same tab as discussed with @wpscholar